### PR TITLE
860 remove angular strap

### PR DESCRIPTION
**Pull request by davidmiller**

### CODE REVIEW

File: elcid/assets/js/elcid/services/records/diagnosis.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ angular.module('opal.records')
         return function(record){
             if(!record.id){
                 if(!record.date_of_diagnosis){
-                    record.date_of_diagnosis = moment()
+                    record.date_of_diagnosis = moment().toDate();
```
Comment (fredkingham) [inline elcid/assets/js/elcid/services/records/diagnosis.js]: this should be a moment right? We use moment's in the records and dates in the forms?

File: elcid/templates/inline_forms/clinical_advice.html [hunk 2]
```diff
@@ -1,69 +1,72 @@
...
-                  <label class="control-label col-sm-3">Discussed with</label>
-                  <div class="col-sm-9">
-        	    <input class="form-control" type="text" ng-model="clinicalAdviceForm.editing.discussed_with">
-                  </div>
-                </div>
-              </form>
+          <div class="form-group">
+            <label class="control-label col-sm-3">Date</label>
+            <div class="col-sm-9">
+              <input type="text" class="form-control" uib-datepicker-popup="'yyyy/MM/dd"
```
Comment (fredkingham) [inline elcid/templates/inline_forms/clinical_advice.html]: do we need this uib datepicker popup definition?

File: elcid/assets/js/elcid/services/records/diagnosis.js (hunk 1 above)
Comment (davidmiller) [inline elcid/assets/js/elcid/services/records/diagnosis.js]: Is that what we do? It seems to work fine testing...

File: elcid/templates/inline_forms/clinical_advice.html (hunk 2 above)
Comment (davidmiller) [inline elcid/templates/inline_forms/clinical_advice.html]: Um not sure...  it was an explicit markup bs-datepicker before...

**Pull request merged**